CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## [10.4.0](https://github.com/theforeman/puppet-pulpcore/tree/10.4.0) (2024-09-04)
+
+[Full Changelog](https://github.com/theforeman/puppet-pulpcore/compare/10.3.0...10.4.0)
+
+**Implemented enhancements:**
+
+- Reuse headers from pulpcore::apache class [\#354](https://github.com/theforeman/puppet-pulpcore/pull/354) ([ekohl](https://github.com/ekohl))
+
+**Fixed bugs:**
+
+- Don't use underscores in HTTP headers [\#357](https://github.com/theforeman/puppet-pulpcore/pull/357) ([evgeni](https://github.com/evgeni))
+- Correct container plugin apache header rewrite [\#356](https://github.com/theforeman/puppet-pulpcore/pull/356) ([ekohl](https://github.com/ekohl))
```

Comment:
technically not a bug in any released version

Reply:
You're right and there was sufficient discussion for those who read the actual PR. I'll skip it.